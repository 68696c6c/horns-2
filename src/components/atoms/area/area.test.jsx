/* global describe, it, expect */
import React from 'react'

import { render } from 'setup-test'

import Area from '.'

describe('Area', () => {
  it('should render as default', () => {
    const { container } = render(<Area>example</Area>)
    expect(container).toMatchSnapshot()
  })
})
