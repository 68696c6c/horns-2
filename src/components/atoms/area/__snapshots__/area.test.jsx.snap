// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Area should render as default 1`] = `
<div>
  <div
    class="css-1dcsceh"
  >
    example
  </div>
</div>
`;
